README.md

```diff
@@ -99,7 +103,7 @@ or
 </project>
 ```
 
-The first method is better because once the property is defined it's value can be used in other places of the build file. For example in `scala-library` dependency version every Scala build should declare. 
+The first method is better because once the property is defined its value can be used in other places of the build file. For example in `scala-library` dependency version every Scala build should declare. 
```

Comment:
```suggestion
The first method is better because once the property is defined its value can be used in other places of the build file. For example in the `scala-library` dependency version every Scala build should declare. 
```